### DISCUSSION TITLE
Is it possible to disable a bunch of plugins when writing a commit message?

### DISCUSSION THREAD

**Discussion by felipesere** [LazyVim/LazyVim#4124] [Q&A]
When writing commit messages (`git commit`, no `-m` or `--message`) its a bit annoying to have auto-complete trigger or get notifications for possible updates for plugins...

Any idea how I could make this happen?

**Comment by folke** [comment]
In your lazy options, you can set `opts.checker.notify = false`

**Comment by dpetka2001** [comment]
For auto-complete you can use `opts.enabled` in `nvim-cmp` spec, which by default is defined [here](https://github.com/hrsh7th/nvim-cmp/blob/d818fd0624205b34e14888358037fb6f5dc51234/lua/cmp/config/default.lua#L10-L16) and you can add to it for example to also check for filetype `gitcommit`, which is the filetype of the buffer when you do `git commit`.
